test/Elastic.Apm.AspNetFullFramework.Tests/DistributedTracingTests.cs

```diff
@@ -152,6 +152,42 @@ public async Task CallSoapRequest()
 			});
 		}
 
+		[AspNetFullFrameworkFact]
+		public async Task CallSoap12Request()
+		{
+			var rootTxData = SampleAppUrlPaths.CallSoapServiceProtocolV1_2;
+			var fullUrl = Consts.SampleApp.RootUrl + "/" + rootTxData.RelativeUrlPath;
+			var action = "Ping";
+
+			var httpContent = new StringContent($@"<?xml version=""1.0"" encoding=""utf-8""?>
+                <soap12:Envelope xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/2001/XMLSchema"" xmlns:soap12=""http://www.w3.org/2003/05/soap-envelope"">
+                  <soap12:Body>
+                    <{action} xmlns=""http://tempuri.org/"" />
+                  </soap12:Body>
+                </soap12:Envelope>", Encoding.UTF8, "text/xml");
+
+			using (var client = new HttpClient())
+			{
+				var request = new HttpRequestMessage()
+				{
+					RequestUri = new Uri(fullUrl),
+					Method = HttpMethod.Post,
+					Content = httpContent
+				};
+
+				client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("text/xml"));
+				request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/soap+xml; charset=utf-8");
```

Comment:
This throws an exception for me: 

```
Unhandled Exception: System.FormatException: The format of value 'application/soap+xml; charset=utf-8' is invalid.
```

I don't think the `charset=utf-8` part is needed.

```suggestion
				request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/soap+xml");
```